addons/website_mail/controllers/main.py

```diff
@@ -87,6 +87,13 @@ def website_message_subscribe(self, id=0, object=None, message_is_follower="on",
         _message_is_follower = message_is_follower == 'on'
         _object = request.registry[object]
 
+        record = _object.browse(cr, uid, _id, context=context).exists()
+        if not record:
+            return False
+
+        _object.check_access_rule(cr, uid, [_id], 'read', context)
+        _object.check_access_rule(cr, uid, [_id], 'read', context)
```

Comment:
It looks like a typo in backport. Please fix.

Reply:
good catch :+1: